### ISSUE TITLE
EB doesn't load as quick

### ISSUE THREAD

**Issue by IndieKA** [Khan/live-editor#482]
I understand that the new error experience just released not too long ago, but it can't keep up with the mistakes I make in my code. You should try fixing that problem. Thanks!

**Comment by kevinbarabash** [comment]
@IndieKA what you mean by "keep up"?  Do you prefer the old error experience?

**Comment by kevinbarabash** [comment]
It seems like this is quite similar to https://github.com/Khan/live-editor/issues/474.  Am I missing something?

**Comment by IndieKA** [comment]
Yes, I hadn't noticed that. What I mean by "keep up" is when I make a error, it takes a while for Oh Noes to actually check what the error is.

**Comment by pamelafox** [comment]
This is how the new Oh Noes works:
- When there's a problem with your code and your cursor is on the same line as the problem, Oh Noes stays little and says "Hmm". 
- When you move to a new line OR you've been idle on that line for a while, Oh Noes turns into big oh noes.
- You can click on the Hm at any time to find out the error.

So, given that description, can you be more specific with your report? Thanks!
